plugins/woocommerce/src/Internal/Orders/OrderAttributionController.php

```diff
@@ -175,7 +188,7 @@ function( $order_id, $order ) {
 	 */
 	private function maybe_set_admin_source( WC_Order $order ) {
 		if ( function_exists( 'is_admin' ) && is_admin() ) {
-			$order->add_meta_data( $this->get_meta_prefixed_field( 'type' ), 'admin' );
+			$order->add_meta_data( $this->get_meta_prefixed_field( 'source_type' ), 'admin' );
```

Comment:
I think this might have snuck in from the previous PR
```suggestion
			$order->add_meta_data( $this->get_meta_prefixed_field( 'type' ), 'admin' );
```

Reply:
OK, now I get the change, but it's not necessary since `OrderAttributionMeta::get_meta_prefixed_field()` specifically changes `type` to `source_type` before prefixing:
https://github.com/woocommerce/woocommerce/blob/cb0c42180a5860e6dd3c78b0ff004b5e9b20d1b8/plugins/woocommerce/src/Internal/Traits/OrderAttributionMeta.php#L161-L164
So passing `source_type` or `source` are essentially the same.

Reply:
Reverted in 4992aaeb65d3b2552bdd7cf4f3979aa429625eeb